modular_skyrat/modules/colony_fabricator/code/design_datums/construction.dm

```diff
@@ -0,0 +1,105 @@
+// This file is going to be just all bitflag additions
+
+/datum/design/apc_board/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/airalarm_electronics/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/airlock_board/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/firealarm_electronics/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/control/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/firelock_board/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/polarizer/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/condenser/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/health_sensor/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/igniter/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/infrared_emitter/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/prox_sensor/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/signaler/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/timer/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/voice_analyzer/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/camera_assembly/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/intercom_frame/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/light_tube/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/light_bulb/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/conveyor_belt/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/conveyor_switch/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
+
+/datum/design/lavarods/New()
+	. = ..()
+	build_type |= COLONY_FABRICATOR
```

Comment:
So we'd have access to the lava-resistant rods from that new fabricator too?

Reply:
Yeah, rapid construction wouldn't be all too useful if you couldn't rapidly construct over lava or plasma after all.

Reply:
Hmmm, I guess now that it's technically available through Bitrunning, it isn't really that much of a huge concern.